trunner/target/armv7a9.py

```diff
@@ -0,0 +1,138 @@
+import subprocess
+import time
+from abc import abstractmethod
+from typing import Callable, Optional
+
+from trunner.config import TestContext
+from trunner.dut import Dut, SerialDut
+from trunner.harness import (
+    HarnessBuilder,
+    PloInterface,
+    PloImageLoader,
+    PloImageProperty,
+    ShellHarness,
+    Rebooter,
+    RebooterHarness,
+    FlashError,
+)
+from trunner.harness import TerminalHarness
+from trunner.host import Host
+from trunner.tools import JLinkGdbServer, Phoenixd
+from trunner.types import TestResult, TestOptions
+from .base import TargetBase, find_port
+
+
+class ARMv7A9TargetRebooter(Rebooter):
+    # TODO add text mode reboot
+
+    def _reboot_soft(self):
+        self._reboot_hard()
+
+    def _reboot_hard(self):
+        self.host.set_power(False)
+        # optimal power off time to prevent sustaining chips, e.g. flash memory, related to #540 issue
+        time.sleep(0.75)
+        self.dut.clear_buffer()
+        self.host.set_power(True)
+        time.sleep(0.05)
+
+    def _set_flash_mode(self, flash):
+        self.host.set_flash_mode(not flash)
+
+
+class ZynqGdbPloLoader(TerminalHarness, PloInterface):
+    def __init__(self, dut: Dut, script: str, cwd: Optional[str] = None):
+        TerminalHarness.__init__(self)
```

Comment:
We are inheriting from TerminalHarness, so we want to call its constructor.

Reply:
I meant that inheritance could be skipped or I don't see sth

Reply:
We say that `ZynqGdbPloLoader` is  a harness and follow its interface and logic. Even if it's not necessary to call its constructor we would like to keep it in case of any change in the future.

Reply:
Ok, I thought so. Thanks for the explanation